sources/packages/api/src-sql/CRAIncomeVerifications/Create-cra-income-verifications.sql

```diff
@@ -0,0 +1,59 @@
+CREATE TABLE IF NOT EXISTS sims.cra_income_verifications (
+  id SERIAL PRIMARY KEY,
+  tax_year SMALLINT NOT NULL,
+  reported_income INT NOT NULL,
+  cra_reported_income INT,
+  date_sent TIMESTAMP WITH TIME ZONE,
+  date_received TIMESTAMP WITH TIME ZONE,
+  file_sent VARCHAR(50),
+  file_received VARCHAR(50),
+  match_status_code VARCHAR(50),
+  request_status_code VARCHAR(50),
+  inactive_code VARCHAR(50),
+  -- Reference Columns
+  application_Id INT UNIQUE NOT NULL REFERENCES sims.applications(id) ON DELETE CASCADE,
+  -- Audit columns
+  created_at timestamp without time zone NOT NULL DEFAULT now(),
+  updated_at timestamp without time zone NOT NULL DEFAULT now(),
+  creator INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+  SET
+    NULL,
+    modifier INT NULL DEFAULT NULL REFERENCES sims.users(id) ON DELETE
+  SET
+    NULL
+);
+
+-- ## Comments
+COMMENT ON TABLE sims.cra_income_verifications IS 'Income verifications that must be performed with CRA.';
+
+COMMENT ON COLUMN sims.cra_income_verifications.id IS 'Auto-generated sequential primary key column';
```

Comment:
As agreed, please add period at the end

Reply:
I already gave up on this because I keep seeing it being missing again and again in the PRs.
I will be doing the change either way 😉

Reply:
Fixed!